addon.xml

```diff
@@ -1,26 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon id="script.service.hyperion-control" name="Hyperion Control" version="19.0.2" provider-name="hyperion-project">
   <requires>
-	<import addon="xbmc.addon"  version="17.0.0"/>
-    <import addon="xbmc.python" version="3.0.0"/>
-	<import addon="script.module.simplejson" version="3.3.0"/>
-	<import addon="script.module.httplib2" version="0.8.0"/>
+	<import addon="xbmc.addon"  version="20.0.0"/>
+    <import addon="xbmc.python" version="3.0.1"/>
+    <import addon="script.module.requests" version="2.3.0"/>
   </requires>
-  <extension point="xbmc.service" library="service.py" start="login">
-    <provides>service</provides>
-  </extension>
+  <extension point="xbmc.service" library="service.py"/>
```

Comment:
I am not sure, should it not be 

`<extension point="xbmc.service" library="addon.py"/>`

I guess there is no service.py? In the other addon this was changed that way...

Reply:
That is the main entry point, and can be named as we want. The important thing is that the script in the root folder is named in the same way.